Shared/Samples/Configure basemap style parameters/README.md

```diff
@@ -0,0 +1,41 @@
+# Configure basemap style parameters
+
+Apply basemap style parameters customization for a basemap, such as displaying all labels in a specific language or displaying every label in their corresponding local language.
+
+![Image of Configure basemap style parameters](configure-basemap-style-parameters.png)
+
+## Use case
+
+When creating an application that is used in multiple countries, basemaps can reflect the languages and cultures of the users' location. For example, if an application user is in Greece, displaying the labels on a basemap in Greek reflects the local language. Customizing the language setting on the basemap can be controlled by an application user (such as by setting preferences), or implicitly managed within the application logic (by querying the locale of the platform running the application).
+
+## How to use the sample
+
+This sample showcases the workflow of configuring basemap style parameters by displaying a basemap with labels in different languages and launches with a `Viewpoint` set over Bulgaria, Greece and Turkey, as they use three different alphabets: Cyrillic, Greek, and Latin, respectively. By default, the `BasemapStyleLanguage` is set to `local` which displays all labels in their corresponding local language. This can be changed to `global`, which displays all labels in English. The `specific` option sets all labels to a selected language and overrides the `BasemapStyleLanguage` settings.
```

Comment:
```suggestion
This sample showcases the workflow of configuring basemap style parameters by displaying a basemap with labels in different languages and launches with a `Viewpoint` set over Bulgaria, Greece, and Turkey, as they use three different alphabets: Cyrillic, Greek, and Latin, respectively. By default, the `BasemapStyleLanguage` is set to `local` which displays all labels in their corresponding local language. This can be changed to `global`, which displays all labels in English. The `specific` option sets all labels to a selected language and overrides the `BasemapStyleLanguage` settings.
```